### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by unibik**
added setup process of tool

## Description

[Provide a brief description of the changes you have made and why they are necessary.]

## Changes Made
[List the changes you have made in bullet points.]

## Checklist
- [ ] I have read and followed the contributing guidelines.
- [ ] I have tested my changes thoroughly and they work as expected.
- [ ] I have added necessary tests for the changes made.
- [ ] I have updated the documentation to reflect the changes made.
- [ ] My code follows the project's coding style and standards.
- [ ] I have added appropriate commit messages and comments for my changes.

## Related Issues
[If your changes relate to any existing issues, please list them here.]

## Additional Notes
[Provide any additional notes or context that may be helpful for the reviewers.]

Feel free to modify and customize this template as needed to fit the specific needs of the GCP Scanner project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,30 @@ gcp-scanner --help
 There is a docker build file if you want to run the scanner from a container:
 `docker build -f Dockerfile -t sa_scanner .`
 
+### Setup of tool(gcp_scanner)
```
Comment (mshudrak) [inline README.md:65]: Hey, thanks for the commit. We already have instruction on how to use the tool in INSTALL.MD. Could you check with that doc and add these instructions there? We can also reference INSTALL.md from here.

**Review by mshudrak — CHANGES_REQUESTED** [review]
Left some suggestions.

**Comment by HasanZaigam** [comment]
please assign me this problem

**Comment by mshudrak** [comment]
We do not assign problem, feel free to just submit PR.

**Comment by mshudrak** [comment]
No activity, closing the bug.

**Pull request closed without merging**